docs/build/web.md

```diff
@@ -2,7 +2,6 @@
 title: Build for web
 parent: Build ORT
 nav_order: 4
-grand_parent: How to
 ---
 
 # Build ONNX Runtime for Web
```

Comment:
maybe clarify that DOCUMENTATION COMING SOON, rather than functionality coming soon

Reply:
Updated and added link to release blog.